app/js/services/situationService.js

```diff
@@ -226,6 +223,19 @@ angular.module('ddsApp').factory('SituationService', function($http, $sessionSto
                 .concat(enfants)
                 .concat(individus.slice(1));
             situation.individus = individus;
+        },
+
+        ressourcesYearMoins2Captured: function(situation) {
+            var ressourcesYM2Names = ['rncRevenusActivite', 'rncAutresRevenus', 'rncPensionsRetraitesRentes', 'fraisReelsDeductibles', 'rncPensionsAlimentaires', 'rncPensionsAlimentairesVersees'];
+            var ressourcesYM2 = [].concat.apply(
+                [],
+                situation.individus.map(function(individu) {
```

Comment:
I don't get it: you concat an empty array with the result of a `map`, which is always an array? Why not simply take the return value immediately?